gitlab/__init__.py

```diff
@@ -1415,6 +1415,25 @@ def createrepositorytag(self, project_id, tag_name, ref, message=None):
         else:
             return False
 
+    def deleterepositorytag(self, project_id, tag_name):
+
```

Comment:
I'm with Fredrik on this one. The docstring should be directly under the functions declaration.